app/main/menus.js

```diff
@@ -17,7 +17,7 @@ const extraMacFileMenus = [{
 const extraFileMenus = [{
   index: 0,
   menu: {
-    label: i18n.t('&New Session Window…'),
+    label: i18n.t('New Session Window…'),
```

Comment:
Maybe does `&` need to be the same as https://github.com/appium/appium-gui-libs/pull/7 ?